src/api/status/public/js/build-log/build-header.js

```diff
@@ -29,6 +29,24 @@ function renderSha(compare, after) {
   buildGitSHA.innerText = after.substring(0, 7);
 }
 
+function renderLoadingAnimation(code) {
+  buildLoadingAnimation.class = 'material-icons';
```

Comment:
This should be [`className`](https://developer.mozilla.org/en-US/docs/Web/API/Element/className).